automated-actions/AWS_RISK_CREDENTIALS_EXPOSED/README.md

```diff
@@ -0,0 +1,173 @@
+```
+Last, deploy the stack with the resulting yaml (`risk_credentials_exposed.output.yaml`) through the CloudFormation Console or command line:
+
+```bash
+aws cloudformation deploy --region $REGION --template-file risk_credentials_exposed.output.yaml --stack-name $STACKNAME --capabilities CAPABILITY_NAMED_IAM
+```
+
+## Testing the Example
+To test the example without exposing an IAM Access Key to a public repository you can simulate the workflow by executing the `ExposedKey` state machine with a set of test json for the event. To do this follow the steps detailed below.
+
+Ensure you have at least one E-mail address [subscribed](http://docs.aws.amazon.com/sns/latest/dg/SubscribeTopic.html) to the `SecurityNotificationTopic` created by the template to receive the notification.
+
+1. Go to the [Identity & Access Management](https://console.aws.amazon.com/iam/home) console
+1. Go to `Users`
+1. Click `Add User` and create a user with programmatic access
+    * For the `User name` put `test-user`
+    * For `Access type` select `Programmatic access`
+    * **Do not set any permissions for the user**
+    * Note the `Access key ID` from the Review page for later use in the test json
+1. Go to the [Step Functions](https://console.aws.amazon.com/states/home?region=us-east-1#/) console
+1. Select the `ExposedKey` state machine
+1. Click `New execution` to create the test execution of the state machine
+    * Copy and paste the json below to the execution json field
+    * Replace `ACCESS_KEY_ID_HERE` with the `Access key ID` noted from the earlier creation of `test-user`
+    * You do not need to enter an execution id as one will be randomly generated by default
+```json
+{
+    "version": "0",
+    "id": "121345678-1234-1234-1234-123456789012",
+    "detail-type": "AWS Health Event",
+    "source": "aws.health",
+    "account": "123456789012",
+    "time": "2016-06-05T06:27:57Z",
+    "region": "us-east-1",
+    "resources": [],
+    "detail": {
+        "eventArn": "arn:aws:health:us-east-1::event/AWS_RISK_CREDENTIALS_EXPOSED_XXXXXXXXXXXXXXXXX",
+        "service": "RISK",
+        "eventTypeCode": "AWS_RISK_CREDENTIALS_EXPOSED",
+        "eventTypeCategory": "issue",
+        "startTime": "Sat, 05 Jun 2016 15:10:09 GMT",
+        "eventDescription": [
+            {
+                "language": "en_US",
+                "latestDescription": "A description of the event will be provided here"
+            }
+        ],
+        "affectedEntities": [
+            {
+                "entityValue": "ACCESS_KEY_ID_HERE"
+            }
+        ]
+    }
+}
+```
+
+You can follow the state machine's progress in the Step Functions console. The summary message for `test-user`will be sent to the e-mail address(es) subscribed to the `SecurityNotificationTopic` SNS Topic.
+* **Note that the summary for this user will likely be empty as it will not have logged any API calls to CloudTrail**
+
```

Comment:
I would also add a step to go back to the IAM console and verify that the Access Key for 'test-user' has actually been removed.